apps/aws-amplify/README.md

```diff
@@ -0,0 +1,81 @@
+This project was bootstrapped with [Create Contentful App](https://github.com/contentful/create-contentful-app).
+
+## How to use
+
+Execute create-contentful-app with npm, npx or yarn to bootstrap the example:
+
+```bash
+# npx
+npx create-contentful-app --typescript
+
+# npm
+npm init contentful-app -- --typescript
+
+# Yarn
+yarn create contentful-app --typescript
+```
+
+## Available Scripts
+
+In the project directory, you can run:
+
+#### `npm start`
+
+Creates or updates your app definition in Contentful, and runs the app in development mode.
+Open your app to view it in the browser.
+
+The page will reload if you make edits.
+You will also see any lint errors in the console.
+
+#### `npm run build`
+
+Builds the app for production to the `build` folder.
+It correctly bundles React in production mode and optimizes the build for the best performance.
+
+The build is minified and the filenames include the hashes.
+Your app is ready to be deployed!
+
+#### `npm run upload`
+
+Uploads the build folder to contentful and creates a bundle that is automatically activated.
+The command guides you through the deployment process and asks for all required arguments.
+Read [here](https://www.contentful.com/developers/docs/extensibility/app-framework/create-contentful-app/#deploy-with-contentful) for more information about the deployment process.
+
+#### `npm run upload-ci`
```

Comment:
Same comment as above, command doesn't exist - instead this should be `npm run deploy`